index.html

```diff
@@ -1138,57 +1138,56 @@ <h1>
 
     <p>
 A <a>DID</a> points to a <a>DID document</a>. <a>DID documents</a> are the
-serialization of the <a href="#data-model"></a>.
+serialization of the data model outlined in Section <a href="#data-model"></a>.
 The following sections define the properties in a <a>DID document</a>,
 including whether these properties are required or optional.
     </p>
 
     <section>
-      <h2>Contexts</h2>
+      <h2>
+Contexts
+      </h2>
 
-        <p>
+      <p>
 When two software systems need to exchange data, they need to use terminology
 and a protocol that both systems understand. The <code>@context</code>
 property ensures that two systems operating on the same DID document are using
 mutually agreed terminology.
-        </p>
+      </p>
 
-        <p>
+      <p>
 <a>DID documents</a> MUST include the <code>@context</code> property.
-        </p>
+      </p>
 
-        <p class="note" title="The JSON-LD Context">
+      <p class="note" title="The JSON-LD Context">
 More information about the
-<a href="https://www.w3.org/TR/json-ld/#the-context">JSON-LD Context</a>
-in general can be found in the [[!JSON-LD]] specification.
-        </p>
-
-        <dl>
-          <dt><dfn>@context</dfn></dt>
-          <dd>
-The value of the <code>@context</code> property MUST be one or more
-<a>URIs</a>, where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. If more than one
-<a>URI</a> is provided, the <a>URIs</a> MUST be interpreted as an ordered set.
-It is RECOMMENDED that dereferencing the <a>URIs</a> results in a document
-containing machine-readable information about the context.
-          </dd>
-        </dl>
-
-      <p>
-Example:
+<a href="https://www.w3.org/TR/json-ld/#the-context">JSON-LD Context</a> in
+general can be found in the [[!JSON-LD]] specification.
       </p>
 
+      <dl>
+        <dt><dfn>@context</dfn></dt>
+        <dd>
+The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
+where the value of the first <a>URI</a> is
+<code>https://www.w3.org/ns/did/v1</code>. If more than one <a>URI</a> is
+provided, the <a>URIs</a> MUST be interpreted as an ordered set. It is
+RECOMMENDED that dereferencing the <a>URIs</a> results in a document containing
```

Comment:
```suggestion
RECOMMENDED that dereferencing each <a>URI</a> result in a document containing
```

Reply:
Updated, but left it as "results". "result" just didn't sound right to me.